lib/screens/country_list_screen.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:ads/screens/country_detail_screen.dart';
+import 'package:ads/models/country.dart';
+import 'package:flutter/material.dart';
+
+class CountryListScreen extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
```

Comment:
instead of referencing Country.countryList everywhere. declare a local variable and save it in that and then use it everywhere